src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3144,7 +3144,7 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
                 application_security_groups=None, ultra_ssd_enabled=None,
                 ephemeral_os_disk=None, ephemeral_os_disk_placement=None,
                 proximity_placement_group=None, aux_subscriptions=None, terminate_notification_time=None,
-                max_price=None, computer_name_prefix=None, orchestration_mode=None, scale_in_policy=None,
+                max_price=None, computer_name_prefix=None, orchestration_mode='Flexible', scale_in_policy=None,
```

Comment:
Could we add the default value to the definition of parameters `--orchestration-mode`, so that customers can also know the default value from the help message of this parameter?